### ISSUE TITLE
Back button causes CommandPanels items to be interactable

### ISSUE THREAD

**Issue by slash1div** [Iridium-Development/IridiumSkyblock#922]
### Steps to Reproduce

1. Install IridiumSkyblock on a server with CommandPanels.
2. Make a commandpanels item that runs a menu command like "island border"
3. Click the back button.
4. Shift click the items
5. Exit the menu then place the item, or drop the items then rejoin
6. After rejoining, you can pick up the dropped items and the placed blocks are still there.

### Expected Behavior

The menu should have the same behavior as before, where clicking or shift clicking interacts with the buttons.

### Actual Behavior

The menu acts like a chest. You can take items and they will function, although moving them around in the inventory or leaving the game with them in your inventory will make them disappear. Placing or dropping the items on the ground, rejoining, then picking them up makes them stay.

### Screenshots

<video src="https://github.com/user-attachments/assets/47010b3f-a5f7-441c-a893-e874233f0208" controls></video>

### Server Software & Minecraft Version

git-Purpur-2176 (MC: 1.20.4)* You are running the latest version

### IridiumSkyblock version

4.0.9.1

### Server logs

No error messages: https://mclo.gs/2Fbxlru

### Additional Comments

Viaversion and Viabackwards have been uninstalled.

Command arguments for the item in the menu are:
```
sudo= island border
cpc
```
Issue will happen regardless of cpc being added.

**Comment by PeachesMLG** [comment]
This would be a bug with CommandPanels, not us. All we do is open the same inventory instance that was open before.

You can either disable this feature in the config (Honestly cant remember if this was configurable or not, but look around)

Or CommandPanels would have to fix this on their side

**Comment by slash1div** [comment]
I was told to open an issue by sh0inx in the Discord as it's an issue that would most likely happen with some other plugins like shops.

Also, the back button is not removable and will stay there even if it's set to air.

**Comment by PeachesMLG** [comment]
Right, if there isnt an option to disable it we can add that fairly easily
